src/components/TopicsList.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { getTopics } from "../api";
+import TopicCard from "./TopicCard";
+
+const TopicsList = () => {
+  const [topics, setTopics] = useState([]);
+
+  useEffect(() => {
+    getTopics().then((topicsData) => {
+      setTopics(topicsData.topic);
+    });
+  }, []);
+
+  return (
+    <ul>
```

Comment:
Great that you're using an API call to populate the topics, please add a[ loading pattern ](https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns)to this page though, while the API is warming up the page looks blank and navigation is confusing